packages/scoutgame/src/points/dividePointsBetweenBuilderAndScouts.ts

```diff
@@ -7,8 +7,8 @@ import { builderPointsShare, scoutPointsShare } from '../builderNfts/constants';
 import { calculateEarnableScoutPointsForRank } from '../points/calculatePoints';
 
 const nftTypeMultipliers: Record<BuilderNftType, number> = {
-  starter_pack: 1,
-  default: 10
+  starter_pack: 0.1,
```

Comment:
i find it easier to think of starter pack's value relative to 'default', versus the other way around. for example if we added a 3rd NFT, it makes more sense if they are all relative to default and not starter_pack